addons/binding/org.openhab.binding.allplay/README.md

```diff
@@ -108,22 +103,22 @@ String All2CoverUrl  "Cover Art URL [%s]"   {channel="allplay:speaker:9fbe37ca-d
 
 demo.sitemap:
 
-```
+```perl
 sitemap demo label="Main Menu"
 {
-		Frame label="All2" {
-			Default item=All2Control
-			Slider item=All2Volume
-			Text item=All2Title
-			Text item=All2Artist
-			Text item=All2State
-		}
+    Frame label="All2" {
+      Default item=All2Control
+      Slider item=All2Volume
+      Text item=All2Title
+      Text item=All2Artist
+      Text item=All2State
+    }
 }
```

Comment:
Indention isn't right here. Please move the closing bracket one line up, then use for spaces per indention level.